### PR TITLE
distrho: 2020-07-14 -> 2021-03-15

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/distrho/default.nix [hunk 1]
```diff
@@ -1,38 +1,55 @@
...
+      libGL
+      libX11
+      libXcursor
+      libXext
+      libXinerama
+      libXrandr
+      libXrender
+      libgcc
+      libglvnd
+      libsecret
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/distrho/default.nix:52]: This formatting is wrong. There should be 2 spaces.

Comment (magnetophon) [reply pkgs/applications/audio/distrho/default.nix]: Done.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118928` at f55466d7 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>distrho</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/audio/distrho/default.nix:93:5:
```
   |
93 |     license = with licenses; [ gpl2 gpl3 gpl2Plus lgpl3 mit ];
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`lgpl3` is a deprecated license, please check if project uses `lgpl3Plus` or `lgpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/audio/distrho/default.nix:93:5:
```
   |
93 |     license = with licenses; [ gpl2 gpl3 gpl2Plus lgpl3 mit ];
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/applications/audio/distrho/default.nix:93:5:
```
   |
93 |     license = with licenses; [ gpl2 gpl3 gpl2Plus lgpl3 mit ];
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/applications/audio/distrho/default.nix [hunk 2]
```diff
@@ -71,9 +89,10 @@ stdenv.mkDerivation rec {
         tal-vocoder-2
         temper
         vex
+        vitalium
         wolpertinger
     '';
-    license = with licenses; [ gpl2 gpl3 gpl2Plus lgpl3 mit ];
+    license = with licenses; [ gpl2Only gpl3Onlu gpl2Plus lgpl2Plus lgpl3Only mit ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/distrho/default.nix:95]:
```suggestion
    license = with licenses; [ gpl2Only gpl3Only gpl2Plus lgpl2Plus lgpl3Only mit ];
```

**Pull request merged**